composer.json

```diff
@@ -1,19 +1,25 @@
 {
     "name": "elife/silex-starter",
     "description": "eLife Silex Starter Project",
+    "config": {
+        "platform": {
+            "php": "7.0.33"
```

Comment:
version available on 16.04

can't specify wildcards or other flags to make patch version dynamic

patch version matters because '7.0' fails the constraint '7.0.8'

Reply:
sounds reasonable but won't it break if 16.04 updates the package? Why are we constraining it?

Reply:
I'm not sure it actually checks the patch version. It could be `7.0.99999` for it all cares.

It is optional, but it's there because I can't get generate a .lock file unless I'm within a VM

Reply:
it will also have to be modified when I upgrade project to 18.04 ... maybe I'll drop it

Reply:
dropped